src/RequestMatcher.php

```diff
@@ -164,6 +164,24 @@ public static function getSiteVariables($site = NULL) {
     return $vars;
   }
 
+  /**
+   * Gets the variant url.
+   *
+   * @param string $variant
+   *   Variant name.
+   *
+   * @return bool|string
+   *   Variant url or bool if variant incorrect.
+   */
+  public function getHostForSiteVariant($variant) {
+    if (!in_array($variant, $this->variants)) {
+      return FALSE;
+    }
+    $site = static::determineActiveSite();
+    $url = getenv('URL_SCHEME') . '://' . $variant . $this->variantSeparator . $site . $this->multisiteDomainSeparator . $this->multisiteDomain;
```

Comment:
not true in every case, check logic of getSiteVariable() method.

Reply:
also this is get HOST - adding the URL_SCHEME must be done in elsewhere. also this variable is not defined to be required in the README..